mock/participants/index.md

```diff
@@ -0,0 +1,32 @@
+# Informations for Members and Participants
+## Menu
+* [Join a group](#) (below)
+* [Invited Experts](#)
+* [Attending Teleconferences](#)
+* [Using Github](#) (separate section)
```

Comment:
Add "Making contributions" section?

Reply:
Great idea! Added it to the menu with a note to continue adding the content later